src/libdredd/src/mutate_visitor.cc

```diff
@@ -391,6 +391,18 @@ void MutateVisitor::HandleExpr(clang::Expr* expr) {
     return;
   }
 
+  clang::Expr::NullPointerConstantKind const kind = expr->isNullPointerConstant(
```

Comment:
Can you add a comment for this block of code to explain what it is trying to avoid being mutated?